docs/development_environment.mdx

```diff
@@ -93,6 +93,11 @@ brew install python3 autoconf ffmpeg
 
 If you encounter build issues with `cryptography` when running the `script/setup` script below, check the cryptography documentation for [installation instructions](https://cryptography.io/en/latest/installation/#building-cryptography-on-macos).
```

Comment:
This sentence links to the exact instructions that you want to add. We use the link because the cryptography instructions will always be up to date.